mat/mul_test.go

```diff
@@ -239,6 +239,10 @@ func (m *basicBanded) TBand() Banded {
 	return TransposeBand{m}
 }
 
+func (m *basicBanded) Trace() float64 {
```

Comment:
Please remove this; the `basicType` types do not satisfy anything except their base method set.